user_guide_src/source/changelogs/v4.3.0.rst

```diff
@@ -43,6 +43,7 @@ Enhancements
 - Added ``spark filter:check`` command to check the filters for a route. See :ref:`Controller Filters <spark-filter-check>` for the details.
 - Now **Encryption** can decrypt data encrypted with CI3's Encryption. See :ref:`encryption-compatible-with-ci3`.
 - Now ``spark routes`` command shows route names. See :ref:`URI Routing <routing-spark-routes>`.
+- ``BaseConnection::escape()`` now excludes the ``RawSql`` data type. This allows passing SQL strings into data. See :ref:`database-queries-db-escape`.
 
```

Comment:
undefined label: database-queries-db-escape

How do I link this? 

See :ref:\`database-queries-db-escape\`.

Reply:
See https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/documentation.rst#references

Reply:
I think I have it fixed..